UserShiftsApiService/UserShiftsApiService/Program.cs

```diff
@@ -64,10 +70,12 @@
     app.UseCors("AllowFront");
 }
 
-app.UseSwagger();
-app.UseSwaggerUI();
 
 app.UseAuthorization();
+app.UseMiddleware<UserIdMiddleware>();
```

Comment:
you only need this middleware in endpoints which assume the FE user accessed. not auth0 webhooks for example